### PR TITLE
Remove onnx-caffe2

### PR DESCRIPTION
**Pull request by yinghai**
Since we have moved `onnx-caffe2` into `caffe2`. Update the documentation here.

### CODE REVIEW

**Comment by yinghai** [comment]
@houseroad

File: docs/source/onnx.rst [hunk 1]
```diff
@@ -79,28 +79,22 @@ Then, you can run::
     # Print a human readable representation of the graph
     onnx.helper.printable_graph(model.graph)
 
-To run the exported script with `caffe2 <https://caffe2.ai/>`_, you will need three things:
+To run the exported script with `caffe2 <https://caffe2.ai/>`_, you will need to install `caffee2`:
```
Comment (houseroad) [inline docs/source/onnx.rst]: Typo, caffe2

File: docs/source/onnx.rst [hunk 2]
```diff
@@ -79,28 +79,22 @@ Then, you can run::
...
 
 1. You'll need an install of Caffe2.  If you don't have one already, Please
```
Comment (houseroad) [inline docs/source/onnx.rst]: Also please remove the numerical bullet

**Review by houseroad — COMMENTED** [review]
Thanks for updating the doc.

Please address the comments.

**Comment by yinghai** [comment]
Addressed comments.

**Review by houseroad — APPROVED** [review]
LGTM, thanks!

**Pull request merged**